client/my-sites/patterns/components/pattern-library/index.tsx

```diff
@@ -359,27 +364,26 @@ export const PatternLibrary = ( {
 					</PatternLibraryBody>
 				) }
 
-				{ isEnabled( 'readymade-templates/showcase' ) && isHomePage && <ReadymadeTemplates /> }
-
 				{ ! isEnabled( 'readymade-templates/showcase' ) && isHomePage && (
 					<PatternsCopyPasteInfo theme="dark" />
 				) }
 
 				{ isHomePage && (
-					<CategoryGallery
-						title={ pageLayoutsHeading }
-						description={ translate(
-							'Start even faster with ready-to-use pages and preassembled patterns. Then tweak the design until it’s just right.'
-						) }
-						categories={ categories?.filter( ( c ) => c.pagePatternCount ) }
-						patternTypeFilter={ PatternTypeFilter.PAGES }
-					/>
+					<>
+						<CategoryGallery
+							title={ pageLayoutsHeading }
+							description={ translate(
+								'Our page layouts are exactly what you need to easily create professional-looking pages using preassembled patterns.'
+							) }
+							categories={ categories?.filter( ( c ) => c.pagePatternCount ) }
+							patternTypeFilter={ PatternTypeFilter.PAGES }
+						/>
+						{ isEnabled( 'readymade-templates/showcase' ) && <ReadymadeTemplates /> }
+					</>
 				) }
-
-				{ isEnabled( 'readymade-templates/showcase' ) && isHomePage && <PatternsCopyPasteInfo /> }
 			</div>
 
-			<PatternsGetStarted theme="dark" />
+			<PatternsGetStarted theme={ isEnabled( 'readymade-templates/showcase' ) ? 'blue' : 'dark' } />
```

Comment:
This can be blue in prod as well.

Reply:
I prefer to not change how production looks until we open our RTs changes.

Reply:
> I prefer to not change how production looks until we open our RTs changes.

Then the same should be applied to the text changes, right?